theme_test_custo/static/src/scss/primary_variables.scss

```diff
@@ -0,0 +1,108 @@
+// -------------------------------------------------------------------------- //
+// PRESETS
+// -------------------------------------------------------------------------- //
+
+$o-website-values-palettes: (
+    (
+        // Colors
+        'color-palettes-name':              'custom-theme',
+
+        // Fonts
+        'font':                             'Source Sans Pro',
+        'headings-font':                    'Poppins',
+        'navbar-font':                      'Source Sans Pro',
+        'buttons-font':                     'Source Sans Pro',
+
+        // Header
+        'header-template':                  'custom-theme',
+        'logo-height':                      4rem,
+
+        // Footer
+        'footer-template':                  'custom-theme',
+    ),
+);
+
+// -------------------------------------------------------------------------- //
+// FONTS
+// -------------------------------------------------------------------------- //
+
+$o-theme-font-configs: (
+    'Source Sans Pro': (
+        'family':                           ('Source Sans Pro', sans-serif),
+        'url':                              'Source+Sans+Pro:300,400,400i,600',
+        'properties' : (
+            'base': (
+                'font-size-base':           1rem,
+            ),
+        ),
+    ),
+    'Poppins': (
+        'family':                           ('Poppins', sans-serif),
+        'url':                              'Poppins:400,500',
+        'properties' : (
+            'base': (
+                'font-size-base':           1rem,
+            ),
+        ),
+    ),
+);
+
+// -------------------------------------------------------------------------- //
+// FONTS
+// -------------------------------------------------------------------------- //
+
+$o-theme-font-configs: (
+    'Source Sans Pro': (
+        'family': ('Source Sans Pro', sans-serif),
+        'url': 'Source+Sans+Pro:300,400,400i,600',
+        'properties' : (
+            'base': (
+                'font-size-base':           1rem,
+            ),
+        )
+    ),
+    'Lobster': (
+        'family': ('Lobster', sans-serif),
+        'url': 'Lobster:400',
+        'properties' : (
+            'base': (
+                'font-size-base':           1rem,
+            ),
+        )
+    ),
+);
```

Comment:
Oups? Duplicated code, `$o-theme-font-configs` copy/pasted twice :grimacing: You can also drop the "properties" key in both fonts